mule-extensions-api-persistence/src/test/resources/property/imported-types-model-property.json

```diff
@@ -3,18 +3,248 @@
     {
       "type": {
         "format": "java",
-        "type": "String",
+        "type": "Object",
```

Comment:
because in order to have a more realistic test case, I've changed the setup from this:
`new ImportedTypesModelProperty(singletonMap(typeLoader.load(String.class), typeLoader.load(Integer.class)));`

to
`new ImportedTypesModelProperty(singletonMap(typeLoader.load(ComplexFieldsType.class), OTHER_EXTENSION_NAME));`